src/transform/src/normalize_lets.rs

```diff
@@ -709,6 +709,8 @@ mod inlining {
                             &expr
                         };
                         match stripped_value {
+                            // TODO: One could imagine CSEing multiple occurrences of a global Get
+                            // to make us read from Persist only once.
```

Comment:
Can you file an issue for this?

Reply:
Yes, done: https://github.com/MaterializeInc/materialize/issues/21145